tools/storybook/package.json

```diff
@@ -11,13 +11,15 @@
 	"author": "Automattic",
 	"license": "GPL-3.0-or-later",
 	"scripts": {
-		"build-storybook": "pnpm build-woocommerce && pnpm build-woocommerce-blocks-storybook && ./import-wp-css-storybook.sh && BABEL_ENV=storybook STORYBOOK=true storybook build -c ./.storybook",
-		"postbuild-storybook": "cp ../../plugins/woocommerce-blocks/storybook/dist/iframe.html ./storybook-static/assets/woocommerce-blocks",
-		"build-woocommerce": "pnpm --filter=@woocommerce/plugin-woocommerce build",
-		"build-woocommerce-blocks-storybook": "pnpm --filter=@woocommerce/block-library storybook:build",
-		"preinstall": "npx only-allow pnpm",
-		"storybook": "pnpm build-woocommerce && ./import-wp-css-storybook.sh && BABEL_ENV=storybook STORYBOOK=true storybook dev -c ./.storybook -p 6007 --ci",
-		"storybook-rtl": "USE_RTL_STYLE=true pnpm storybook"
+		"build": "pnpm --if-present --workspace-concurrency=Infinity --stream --filter=\"$npm_package_name...\" '/^build:project:storybook$/'",
```

Comment:
Only trigger the `build:project:storybook` as we don't actually need to build the other dependencies, storybook does that for us.

Reply:
Rather than only building `build:project:storybook` (the regex is unnecessary here too), we should remove the `--filter` option that way it will only run on the current package. With that done, we can also remove `--if-present` since we _know_ the script is present here.